xamarin/realm-tutorial-dotnet/LoginPage.xaml.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using AsyncTask = System.Threading.Tasks.Task;
+using Realms.Sync;
+using Xamarin.Forms;
+
+namespace RealmTemplateApp
+{
+    public partial class LoginPage : ContentPage
+    {
+        private string email;
+        private string password;
+        private bool login;
+
+        public LoginPage()
+        {
+            InitializeComponent();
+        }
+
+        protected override void OnAppearing()
+        {
+            base.OnAppearing();
+            txtEmail.Text = "";
+            txtPassword.Text = "";
+            login = false;
+        }
+
+        async void Main_Button_Clicked(object sender, EventArgs e)
+        {
+            if (login)
+            {
+                await DoLogin();
+            }
+            else
+            {
+                await RegisterUser();
+            }
+        }
+
+        private async AsyncTask DoLogin()
```

Comment:
This is the default behavior of VS when it creates event handlers (it doesn't add the access modifier). I've added them to avoid further back-and-forth, but do not think the consistency here is necessarily "right". ¯\_(ツ)_/¯